Avoid a crash caused by UB (use of uninitialized value).

Initialize a value to NULL and backport code handling this case from main.
This won't hurt and avoids the crash, although it's probably not the correct
fix.

Upstream issue: https://github.com/fvwmorg/fvwm3/issues/659

--- modules/FvwmIconMan/x.c.orig	2022-09-21 21:44:02 UTC
+++ modules/FvwmIconMan/x.c
@@ -741,6 +741,7 @@ void X_init_manager (int man_id)
   if (man->geometry_str) {
     fscreen_scr_arg arg;
     arg.mouse_ev = NULL;
+    arg.name = NULL;
 
     geometry_mask = FScreenParseGeometryWithScreen(
       man->geometry_str, &man->geometry.x, &man->geometry.y,
